### PR TITLE
Youtubedown.js packager

### PR DESCRIPTION
**Pull request by simonhong**
npm run generate-youtubedown -- --key "private key file path"
npm run package-youtubedown -- --binary "chrome bin path" --key "private key file path" 
npm run upload-youtubedown-component -- --crx-directory ./build/youtubedown/output

### CODE REVIEW

File: scripts/packageYoutubedown.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+      })
+      .pipe(fs.createWriteStream(destPath))
+  })
+}
+
+async function downloadLatestYoutubedown (bucketUrl) {
+  // TODO(bsclifton): doesn't handle errors (ex: 403)
+  // script is currently getting a 403 when fetched :(
+  // Because of 403 from 'https://www.jwz.org/hacks/youtubedown.js', we copied
+  // youtubedown.js to s3 bucket.
```
Comment (simonhong) [inline scripts/packageYoutubedown.js:95]: @bsclifton As you wrote, I also can't get script from jwz.org.
As a workaround, bucket hosts it instead.

Comment (bsclifton) [reply scripts/packageYoutubedown.js:95]: I think we can supply a user-agent and it should download just fine. Maybe we can try a few things with `curl` - it reproduces easily with that

Comment (bsclifton) [reply scripts/packageYoutubedown.js:95]: See https://bravesoftware.slack.com/archives/CKJCXTP9B/p1596702301074300

Comment (simonhong) [reply scripts/packageYoutubedown.js]: @bsclifton  Works perfect with UA. Thanks!

**Comment by simonhong** [comment]
Below two prod & dev Job works well with this branch.
https://ci.brave.com/view/all-extensions/job/brave-core-ext-youtubedown-component-upload/
https://ci.brave.com/view/all-extensions/job/brave-core-ext-youtubedown-component-upload-dev/

**Review by bsclifton — APPROVED** [review]
Nice work 😄

**Pull request merged**